bigflow/workflow.py

```diff
@@ -417,3 +420,55 @@ def _parse_runtime_str(runtime: str) -> dt.datetime:
         except ValueError:
             pass
     raise ValueError("Unable to parse 'runtime' %r" % runtime)
+
+
+def load_workflows_in_packages(
+        project_dir: Path,
+        packages: List[str]
+) -> Iterator[Any]:
+    """
+    Imports modules from specified packages and yields elements of type bf.Workflow
+    """
+    if not packages:
+        raise ValueError('At least one package required')
+
+    # TODO move walk_workflows from cli to workflows
+    from bigflow.cli import walk_workflows
+    return (
+        w
+        for p in packages if '.' not in p
+        for w in walk_workflows(project_dir / p)
+    )
+
+
+def verify_workflow(workflow_id: str) -> None:
+    """
+    Verifies that workflow with id workflow_id is present and can be correctly loaded, raises exception otherwise.
```

Comment:
please, pass all kinds of documentation through grammarly, it's an easy way of fixing some basic errors, like missing a/an/the